theano/sandbox/cuda/cula.py

```diff
@@ -0,0 +1,127 @@
+import theano
+from theano.sandbox.cuda.type import CudaNdarrayType
+from theano.sandbox.cuda import GpuOp
+
+from theano.sandbox.cuda.basic_ops import as_cuda_ndarray_variable
+from theano.sandbox.cuda import cuda_ndarray
+
+dimshuffle = cuda_ndarray.cuda_ndarray.dimshuffle
+
+try:
+    from scikits.cuda import cula
+    cula_available = True
+except (ImportError, OSError):
+    pass
+
+cula_initialized = False
+
+
+class GpuSolve(GpuOp):
+    """
+    CULA GPU solver OP.
+
+    :param trans: Whether to take the transpose of the input matrix
+    or not.
+    """
+    __props__ = ('trans')
```

Comment:
You need to set it up as a 1-tuple: `__props__ = ('trans',)`, otherwise it tries to find each letter as a different attribute.